### DISCUSSION TITLE
Activity Event

### DISCUSSION THREAD

**Discussion by IamHomen** [Sketchware-Pro/Sketchware-Pro#854] [Q&A]
How to add Activity event to sketchware dev tools

@Override
    protected void onPostExecute(String ress) {
        %s
        }

even adding this, still no value on source code

**Comment by DevVigilante** [comment]
> How to add Activity event to sketchware dev tools
> 
> @OverRide protected void onPostExecute(String ress) { %s }
> 
> even adding this, still no value on source code

Just send screen shot of property setting.

**Comment by IamHomen** [comment]
Activity Event

**Comment by DevVigilante** [comment]
> Activity Event

Yes,
Eg. where you created event.

**Comment by IamHomen** [comment]
i added the onPostExecute but returned this error

----------
1. ERROR in
/storage/emulated/0/.sketchware/mysc/674/app/src/main/java/com/mrcain/animeyay/MainActivity.java
(at line 310)
 protected void onPostExecute(String _result) {
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
The method onPostExecute(String) of type MainActivity must override or
implement a supertype method
----------
1 problem (1 error)

**Comment by DevVigilante** [comment]
> i added the onPostExecute but returned this error
> […](#)
> ---------- 1. ERROR in /storage/emulated/0/.sketchware/mysc/674/app/src/main/java/com/mrcain/animeyay/MainActivity.java (at line 310) protected void onPostExecute(String _result) { ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ The method onPostExecute(String) of type MainActivity must override or implement a supertype method ---------- 1 problem (1 error)
> On Sat, Apr 22, 2023, 10:17 AM TechnicalStudioDeveloper < ***@***.***> wrote: Activity Event Yes, Eg. where you created event. — Reply to this email directly, view it on GitHub <[#847 (comment)](https://github.com/Sketchware-Pro/Sketchware-Pro/issues/847#issuecomment-1518473404)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AL663BEVYFFQCBA7BJH37LDXCM5RZANCNFSM6AAAAAAXHNOW3E> . You are receiving this because you authored the thread.Message ID: ***@***.***>

![Screenshot_2023-04-22-07-44-48-042_com sketchware remod](https://user-images.githubusercontent.com/96680248/233758177-6a8f9e16-9038-4924-b1a3-08960f0079bd.jpg)
Send screenshot like above.

**Comment by IamHomen** [comment]
example for onPostExecute(String _res)

**Comment by DevVigilante** [comment]
> example for onPostExecute(String _res) On Sat, Apr 22, 2023, 10:53 AM TechnicalStudioDeveloper < ***@***.***> wrote:
> […](#)
> i added the onPostExecute but returned this error … <#m_7134392686995987999_> ---------- 1. ERROR in /storage/emulated/0/.sketchware/mysc/674/app/src/main/java/com/mrcain/animeyay/MainActivity.java (at line 310) protected void onPostExecute(String _result) { ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ The method onPostExecute(String) of type MainActivity must override or implement a supertype method ---------- 1 problem (1 error) On Sat, Apr 22, 2023, 10:17 AM TechnicalStudioDeveloper < *@*.*> wrote: Activity Event Yes, Eg. where you created event. — Reply to this email directly, view it on GitHub <#847 (comment) <[#847 (comment)](https://github.com/Sketchware-Pro/Sketchware-Pro/issues/847#issuecomment-1518473404)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/AL663BEVYFFQCBA7BJH37LDXCM5RZANCNFSM6AAAAAAXHNOW3E <https://github.com/notifications/unsubscribe-auth/AL663BEVYFFQCBA7BJH37LDXCM5RZANCNFSM6AAAAAAXHNOW3E> . You are receiving this because you authored the thread.Message ID: @.*> [image: Screenshot_2023-04-22-07-44-48-042_com sketchware remod] <https://user-images.githubusercontent.com/96680248/233758177-6a8f9e16-9038-4924-b1a3-08960f0079bd.jpg> Send screenshot like above. — Reply to this email directly, view it on GitHub <[#847 (comment)](https://github.com/Sketchware-Pro/Sketchware-Pro/issues/847#issuecomment-1518482150)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AL663BDBOWTKDPYV24V7OK3XCNB3ZANCNFSM6AAAAAAXHNOW3E> . You are receiving this because you authored the thread.Message ID: ***@***.***>

😫😫😫😩

**Comment by IamHomen** [comment]
i added that code & recive that error thats why im asking for correct code!

**Comment by kuskebabi** [comment]
> i added the onPostExecute but returned this error
> […](#)
> ---------- 1. ERROR in /storage/emulated/0/.sketchware/mysc/674/app/src/main/java/com/mrcain/animeyay/MainActivity.java (at line 310) protected void onPostExecute(String _result) { ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ The method onPostExecute(String) of type MainActivity must override or implement a supertype method ---------- 1 problem (1 error)
> On Sat, Apr 22, 2023, 10:17 AM TechnicalStudioDeveloper < ***@***.***> wrote: Activity Event Yes, Eg. where you created event. — Reply to this email directly, view it on GitHub <[#847 (comment)](https://github.com/Sketchware-Pro/Sketchware-Pro/issues/847#issuecomment-1518473404)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AL663BEVYFFQCBA7BJH37LDXCM5RZANCNFSM6AAAAAAXHNOW3E> . You are receiving this because you authored the thread.Message ID: ***@***.***>

add `@Override` before the void of the event

**Comment by IamHomen** [comment]
i have, and still got the same error
